lnrpc/routerrpc/router.proto

```diff
@@ -490,6 +490,120 @@ message BuildRouteResponse {
     lnrpc.Route route = 1;
 }
 
+
+message SubscribeHtlcEventsRequest{}
+
+message HtlcEvent{
+    /**
+    IncomingChannel is the channel that the incoming htlc arrived at our node
+    on. This value is zero for sends.
+    */
+    uint64 incoming_channel = 1 [json_name="incoming_channel"];
```

Comment:
i might suggest making this `incoming_channel_id` or `incoming_chan_id` since we typically have the `_id` prefix elsewhere in the rpc interface, similar comment for `outgoing_channel`